packages/x-charts-vendor/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@mui/x-charts-vendor",
+  "version": "7.8.0",
+  "description": "Vendored dependencies for MUI charts",
+  "author": "MUI Team",
+  "main": "./index.js",
+  "keywords": [
+    "data visualization",
+    "React",
+    "d3",
+    "charting"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mui/mui-x"
+  },
+  "license": "MIT AND ISC",
+  "exports": {
+    "./package.json": "./package.json",
+    "./*": {
+      "types": "./*.d.ts",
+      "import": "./es/*.js",
+      "default": "./lib/*.js"
+    }
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.24.7",
+    "d3-color": "^3.1.0",
+    "d3-delaunay": "^6.0.4",
+    "d3-interpolate": "^3.0.1",
+    "d3-scale": "^4.0.2",
+    "d3-shape": "^3.2.0",
+    "d3-time": "^3.1.0",
+    "@types/d3-color": "^3.1.3",
+    "@types/d3-delaunay": "^6.0.4",
+    "@types/d3-interpolate": "^3.0.4",
+    "@types/d3-scale": "^4.0.8",
+    "@types/d3-shape": "^3.1.6",
+    "@types/d3-time": "^3.0.3",
+    "robust-predicates": "^3.0.2",
+    "delaunator": "^5.0.1"
+  },
+  "devDependencies": {
+    "d3-format": "^3.1.0",
+    "d3-time-format": "^4.1.0",
+    "d3-path": "^3.0.1",
+    "d3-array": "^3.1.6",
+    "@types/d3-format": "^3.0.4",
+    "@types/d3-time-format": "^4.0.3",
+    "@types/d3-path": "^3.0.1",
+    "@types/d3-array": "^3.0.3",
+    "internmap": "^2.0.3",
+    "execa": "^6.1.0",
```

Comment:
Have you tried using a newer version of `execa`?
This installs yet another major version of it—we already have 3 different versions installed. 🙈 
P.S. This as well as `rimraf` also end up in the commited files, but I suspect that they are only needed for building the lib and could probably be also omitted from Git and the `vendor` package itself. 🤔 🤷

Reply:
Yeah I would assume all of `@babel/runtime`, `execa`, `internmap` and `rimraf` don't need to be built and pushed/bundled 🤔